.github/workflows/coverity.yml

```diff
@@ -0,0 +1,25 @@
+name: coverity-scan
+
+on:
+  schedule:
+    - cron: '0 0 * * *' # Once every day at 00:00 UTC
+env:
+  CC: gcc
+  ARCH: x86_64
+  CONTAINER_NAMESPACE: ghcr.io/opendataplane/odp-docker-images
+  OS: ubuntu_20.04
+  COVERITY_EMAIL: odp@lists.opendataplane.org
+  COVERITY_PROJECT: odp
+
+jobs:
+  Coverity-analysis:
+    runs-on: ubuntu-18.04
+    steps:
+      - uses: actions/checkout@v2
+      - run: sudo docker run -i -v `pwd`:/odp --privileged --shm-size 8g
+               -e CC="${CC}" -e GITHUB_SHA="${GITHUB_SHA}"
+               -e COVERITY_TOKEN="${{ secrets.COVERITY_SCAN_TOKEN }}"
```

Comment:
To be consistent with the odp-docker-images repo, the secret could be named `COVERITY_TOKEN`.

Reply:
Will do.